docs/pages/components/message-box.md

```diff
@@ -0,0 +1,440 @@
+---
+title: Message Box
+id: message-box
+keywords: message message-box
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/message-box.html
+folder: components
+summary:
+---
+
+Message Box is used to display simple messages to the user. These messages could be Standard (Default), Confirmation, Error, Success, Warning and Information (Neutral). The Message Box component inherits the look and the behaviour of the [Dialog]({{site.baseurl}}/components/dialog.html) component. 
```

Comment:
> The Message Box component inherits the look and **the behaviour** of the Dialog

I'm not sure about that part. From my understanding Message Box is a simplified version of the Dialog + semantic states that makes a perfect fit for displaying alerts/confirmation widows etc. I don't think it will provide such features as dragging/resizing/responsiveness. Current description shows Message Box as an extension of the Dialog.

Reply:
perhaps "...a subset of it's behavior..."